src/pages/DelegatoryValidator/Components/IntervalBar.tsx

```diff
@@ -0,0 +1,87 @@
+import {Stack, Typography, useTheme} from "@mui/material";
+import moment from "moment";
+import React from "react";
+import {grey} from "../../../themes/colors/aptosColorPalette";
+import {parseTimestamp, timestampDisplay} from "../../utils";
+import {
+  BAR_BACKGROUND_COLOR,
+  BAR_COLOR,
+} from "../../Validators/Components/Epoch";
+
+export function IntervalBarTimeDuration({timestamp}: {timestamp: number}) {
```

Comment:
looks great! wondering if we can modularize the reusable part from `EpochIntervalBar` and use it in both here and the epoch bar.